monai/transforms/io/array.py

```diff
@@ -31,38 +32,47 @@
 
 class LoadImage(Transform):
     """
-    Load image file or files from provided path based on reader, default reader is ITK.
-    All the supported image formats of ITK:
-    https://github.com/InsightSoftwareConsortium/ITK/tree/master/Modules/IO
+    Load image file or files from provided path based on reader, default reader is Nibabel.
     Automatically choose readers based on the supported suffixes and in below order:
     - User specified reader at runtime when call this loader.
     - Registered readers from the first to the last in list.
-    - Default ITK reader.
+    - Default Nibabel reader.
 
     """
 
     def __init__(
         self,
-        reader: Optional[ImageReader] = None,
+        reader: Optional[Union[ImageReader, str]] = None,
         image_only: bool = False,
         dtype: np.dtype = np.float32,
+        *args,
+        **kwargs,
     ) -> None:
         """
         Args:
             reader: register reader to load image file and meta data, if None, still can register readers
-                at runtime or use the default ITK reader.
+                at runtime or use the default NIbabel reader. If a string of reader name provided, will construct
+                a reader object with the `*args` and `**kwargs` parameters, supported reader name: "NibabelReader",
+                "PILReader", "ITKReader", "NumpyReader"
             image_only: if True return only the image volume, otherwise return image data array and header dict.
             dtype: if not None convert the loaded image to this data type.
+            args: additional parameters for reader if providing a reader name.
+            kwargs: additional parameters for reader if providing a reader name.
 
         Note:
             The transform returns image data array if `image_only` is True,
             or a tuple of two elements containing the data array, and the meta data in a dict format otherwise.
 
         """
-        self.default_reader: ITKReader = ITKReader()
+        self.default_reader: NibabelReader = NibabelReader()
         self.readers: List[ImageReader] = list()
         if reader is not None:
+            if isinstance(reader, str):
+                reader = eval(reader + "(*args, **kwargs)")
```

Comment:
this will introduce some security issues, please change self.readers to a dictionary: `self.readers: Dict[str, ImageReader] = {}`, then we can create a reader instance with:
```py
ReaderClass = self.readers[reader]
reader = ReaderClass(*args, **kwargs)
```

Reply:
Sure, updated in the latest commit.
Thanks.